dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNBusInterfaceIIDM.cpp

```diff
@@ -127,8 +129,10 @@ BusInterfaceIIDM::getComponentVarIndex(const std::string& varName) const {
 
 void
 BusInterfaceIIDM::exportStateVariablesUnitComponent() {
-  busIIDM_.setV(getStateVarV());
-  busIIDM_.setAngle(getStateVarAngle());
+  if (!std::isnan(busIIDM_.getV()) || !doubleEquals(getStateVarV(), 0.)) {
```

Comment:
Are we sure that this will not be a problem ? Like at init value is nan so the value is never updated

Reply:
You forget the `||` !
You should read: if internal V has been previously setted **OR** state variable V has been setted with a non-zero value elsewhere.

Reply:
I mean what if the value was never set ? In that the value will never be updated

Reply:
If the internal V was never set and state variable V was never set too, internal V will never be updated then state variable is considered useless and so not exported.
That's the goal of this PR.

Reply:
OK I didn't get it.